### PR TITLE
Ft/shipping return policy & FAQ

### PR DESCRIPTION
**Pull request by nikunjsdev**
shipping

### CODE REVIEW

File: src/views/More/ShippingReturnPolicy/Page.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import * as React from "react";
+import PageHeader from "../../../components/Header/PageHeader";
+import { Expander } from "@components/molecules/Expander";
+import { list } from "./mock";
+import { useState } from "react";
+import { findIndex } from "lodash";
+
+export const Page = (props: any) => {
+  const [listOfItems, showContent] = useState({ value: list });
```
Comment (zshanhui) [inline src/views/More/ShippingReturnPolicy/Page.tsx]: I think you just need to store the opened/selected index here.

File: src/views/More/ShippingReturnPolicy/Page.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    const toggleItem = listOfItems.value[index];
+    if (alreadyOpenItemIndex > -1 && !expanded) {
+      listOfItems.value.splice(alreadyOpenItemIndex, 1, {
+        ...listOfItems.value[alreadyOpenItemIndex],
+        expanded: false,
+      });
+    }
+    listOfItems.value.splice(index, 1, { ...toggleItem, expanded: !expanded });
+    showContent({ value: listOfItems.value });
+  };
```
Comment (zshanhui) [inline src/views/More/ShippingReturnPolicy/Page.tsx]: You can just store the selected index to simplify this logic.

File: src/views/More/ShippingReturnPolicy/Page.tsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        handleClick={handleClick}
+      />
+      <div>
+        <div className="wrapper-header">Shipping & Return Policy</div>
+        <div className="shipping-return-wrapper">
+          {listOfItems.value.map((item, index) => (
+            <Expander
+              title={item.title}
+              content={item.content}
+              expanded={item.expanded}
```
Comment (zshanhui) [inline src/views/More/ShippingReturnPolicy/Page.tsx]: Then here, you can just compare `selectedIndex === index`

File: src/views/More/ShippingReturnPolicy/mock.ts [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    expanded: false,
```
Comment (zshanhui) [inline src/views/More/ShippingReturnPolicy/mock.ts]: I don't think we need the `expanded` flag, since it's closed by default.

**Review by zshanhui — CHANGES_REQUESTED** [review]
@nikunjsdev 
Seems like there's a chance to simplify some of the logic. Let me know if you have any questions or need me to clarify.

File: src/views/More/ShippingReturnPolicy/Page.tsx (hunk 1 above)
Comment (nikunjsdev) [reply src/views/More/ShippingReturnPolicy/Page.tsx]: @adrienshen You are right
Done changes.

File: src/views/More/ShippingReturnPolicy/Page.tsx (hunk 2 above)
Comment (nikunjsdev) [reply src/views/More/ShippingReturnPolicy/Page.tsx]: Done

File: src/views/More/ShippingReturnPolicy/Page.tsx (hunk 3 above)
Comment (nikunjsdev) [reply src/views/More/ShippingReturnPolicy/Page.tsx]: Done

File: src/views/More/ShippingReturnPolicy/mock.ts (hunk 4 above)
Comment (nikunjsdev) [reply src/views/More/ShippingReturnPolicy/mock.ts]: Done

**Review by zshanhui — DISMISSED** [review]
@nikunjsdev 
Looks good to merge!

**Comment by zshanhui** [comment]
@nikunjsdev 
Looks like there's a tslint error.

```
#!/bin/bash -eo pipefail
npm run tslint

> saleor-site@0.7.1 tslint /home/circleci/app
> tslint 'src/**/*.ts?(x)'


Could not find implementations for the following rules specified in the configuration:
    prettier
Try upgrading TSLint and/or ensuring that you have all necessary custom rules installed.
If TSLint was recently upgraded, you may have old rules configured which need to be cleaned up.
        
Warning: The 'no-chainable-assignment' rule requires type information.
Warning: The 'no-chainable-arguments' rule requires type information.

ERROR: src/@next/components/molecules/Expander/stories.tsx:10:25 - block is empty
ERROR: src/views/More/ShippingReturnPolicy/Page.tsx:5:1 - Multiple imports from 'react' can be combined into one.
ERROR: src/views/More/ShippingReturnPolicy/Page.tsx:18:19 - Expected property shorthand in object literal ('{index}').

npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! saleor-site@0.7.1 tslint: `tslint 'src/**/*.ts?(x)'`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the saleor-site@0.7.1 tslint script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/circleci/.npm/_logs/2020-07-09T20_51_32_917Z-debug.log

Exited with code exit status 2
```

Can you make sure to run `npm run tslint` **before** you push each commit?

**Comment by nikunjsdev** [comment]
> @nikunjsdev
> Looks like there's a tslint error.
> 
> ```
> #!/bin/bash -eo pipefail
> npm run tslint
> 
> > saleor-site@0.7.1 tslint /home/circleci/app
> > tslint 'src/**/*.ts?(x)'
> 
> 
> Could not find implementations for the following rules specified in the configuration:
>     prettier
> Try upgrading TSLint and/or ensuring that you have all necessary custom rules installed.
> If TSLint was recently upgraded, you may have old rules configured which need to be cleaned up.
>         
> Warning: The 'no-chainable-assignment' rule requires type information.
> Warning: The 'no-chainable-arguments' rule requires type information.
> 
> ERROR: src/@next/components/molecules/Expander/stories.tsx:10:25 - block is empty
> ERROR: src/views/More/ShippingReturnPolicy/Page.tsx:5:1 - Multiple imports from 'react' can be combined into one.
> ERROR: src/views/More/ShippingReturnPolicy/Page.tsx:18:19 - Expected property shorthand in object literal ('{index}').
> 
> npm ERR! code ELIFECYCLE
> npm ERR! errno 2
> npm ERR! saleor-site@0.7.1 tslint: `tslint 'src/**/*.ts?(x)'`
> npm ERR! Exit status 2
> npm ERR! 
> npm ERR! Failed at the saleor-site@0.7.1 tslint script.
> npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
> 
> npm ERR! A complete log of this run can be found in:
> npm ERR!     /home/circleci/.npm/_logs/2020-07-09T20_51_32_917Z-debug.log
> 
> Exited with code exit status 2
> ```
> 
> Can you make sure to run `npm run tslint` **before** you push each commit?

Sure, Now time I will run npm run tslint before push anything.

**Pull request merged**